packages/groq-builder/src/commands/conditional.test.ts

```diff
@@ -68,13 +68,13 @@ describe("conditional", () => {
   it("the query should look correct", () => {
     expect(qAll.query).toMatchInlineSnapshot(
       `
-      "*[_type == \\"variant\\"] {
+      "*[_type == "variant"] {
```

Comment:
Almost all of the snapshot changes are because v1.0 no longer escapes quotes. More info [here](https://vitest.dev/guide/migration.html#snapshots-update-3961)

Reply:
Oh this makes me really happy. It's so much easier to read!